virtool/analyses/format.py

```diff
@@ -215,10 +215,10 @@ def calculate_median_depths(document: dict) -> dict:
     return output.getvalue()
 
 
-async def format_analysis_to_csv(db, settings, document):
+async def format_analysis_to_csv(app, document):
     depths = calculate_median_depths(document)
 
-    formatted = await format_analysis(db, settings, document)
+    formatted = await format_analysis(app, document)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'document' in function call](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4878432)